quantgov/corpora/__init__.py

```diff
@@ -5,5 +5,7 @@
     FlatFileCorpusDriver,
     RecursiveDirectoryCorpusDriver,
     NamePatternCorpusDriver,
-    IndexDriver
+    IndexDriver,
```

Comment:
@OliverSherouse Is there a reason this is not called `IndexCorpusDriver`, to follow the pattern above?

Reply:
Nope. Let's add that as a bug and rename for 1.0

Reply:
Added as #46